ArcGISRuntimeSDKQt_QMLSamples/Layers/BrowseOGCAPIFeatureService/BrowseOGCAPIFeatureService.qml

```diff
@@ -0,0 +1,203 @@
+                    Layout.columnSpan: 2
+                    Layout.fillWidth: true
+                }
+                TextField {
+                    id: serviceURLBox
+                    text: serviceURL
+                    Layout.fillWidth: true
+                    selectByMouse: true
+                }
+                Button {
+                    id: connectButton
+                    text: "Load service"
+                    onClicked: {
+                        serviceURL = serviceURLBox.text
+                        loadFeatureService(serviceURL)
+                    }
+                }
+                ComboBox {
+                    id: featureCollectionListComboBox
+                    Layout.columnSpan: 2
+                    Layout.fillWidth: true
+                    Layout.fillHeight: true
+                    textRole: "title"
+                }
+                Button {
+                    id: loadLayerButton
+                    text: "Load selected layer"
+                    onClicked: loadFeatureCollection(featureCollectionListComboBox.currentIndex)
+                    Layout.columnSpan: 2
+                    Layout.fillWidth: true
+                }
+            }
+        }
+        // Pop-up error message box
+        MessageDialog {
+            id: errorMessageBox
+            title: "Error message!"
+            text: errorMessage
+            icon: StandardIcon.Warning
+            visible: errorMessage === "" ? false : true
+            onAccepted: {
+                errorMessage = "";
+            }
+        }
+    }
+
+    function loadFeatureService(currentUrl) {
+        // Create feature service object and assign to featureService property
+        root.featureService = ArcGISRuntimeEnvironment.createObject("OgcFeatureService", {url: currentUrl}, map);
+
+        // Connect loaded signal to updateList() function
+        root.featureService.loadStatusChanged.connect(checkIfServiceLoaded);
+
+        // Load feature service
+        featureService.load();
+    }
+
+    function handleError(error) {
+        if (!error.additionalMessage) {
+            errorMessage = error.message;
+        }
+        else {
+            errorMessage = error.message + "\n" + error.additionalMessage;
+        }
+    }
+
+    function checkIfServiceLoaded() {
+        if (root.featureService.loadStatus === Enums.LoadStatusFailedToLoad) {
+            handleError(root.featureService.loadError);
+            return;
+        }
+        else if (root.featureService.loadStatus !== Enums.LoadStatusLoaded) {
+            return;
+        }
+        updateListInInterface();
+    }
+
+    function updateListInInterface() {
+        // Assign featureCollectionInfos to featureCollectionInfos property
+        let featureCollectionInfos = root.featureService.serviceInfo.featureCollectionInfos;
+
+        // Define ComboBox model using list
+        featureCollectionListComboBox.model = featureCollectionInfos;
```

Comment:
I think this could maybe be a binding that you set up when you declare the combobox?

```
        model: root.featureService.serviceInfo.featureCollectionInfos;
```

might need some ternaries in there to handle the service being null etc...